### PR TITLE
Fix broken links and describe new FileManager API

### PR DESCRIPTION
**Pull request by AnnaKondratova**

### CODE REVIEW

File: api-reference/10 UI Widgets/dxFileManager/1 Configuration/upload/chunkSize.md [hunk 1]
```diff
@@ -5,7 +5,103 @@ default: 200000
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies a chunk size in bytes.
+
+---
+
+The FileManager can upload large files in parts. The widget divides a file into parts and sends them in separate requests.
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxFileManager/1 Configuration/upload/chunkSize.md]: > The FileManager can upload large files in parts. The widget divides a file into parts

Sounds repetitive to me.

```suggestion
The **FileManager** can divide a large file in parts and upload them in separate requests. 
```

File: api-reference/10 UI Widgets/dxFileManager/1 Configuration/upload/chunkSize.md [hunk 2]
```diff
@@ -5,7 +5,103 @@ default: 200000
...
+
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <dx-file-manager id="fileManager">
+        <dxo-upload [chunkSize]="500000"></dxo-upload>
+        <!-- ... -->
+    </dx-file-manager>
+
+    <!-- tab: app.component.ts -->
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxFileManager/1 Configuration/upload/chunkSize.md]: This file can be skipped.

File: api-reference/10 UI Widgets/dxFileManager/1 Configuration/upload/chunkSize.md [hunk 3]
```diff
@@ -5,7 +5,103 @@ default: 200000
...
+    
+    import 'devextreme/dist/css/dx.common.css';
+    import 'devextreme/dist/css/dx.light.css';
+
+    import FileManager, { Upload } from 'devextreme-react/file-manager';
+    
+    class App extends React.Component {
+        render() {
+            return (
+                <FileManager >
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxFileManager/1 Configuration/upload/chunkSize.md]:
```suggestion
                <FileManager>
```

File: api-reference/10 UI Widgets/dxFileManager/5 File System Providers/Custom/1 Configuration/abortFileUpload.md [hunk 4]
```diff
@@ -7,13 +7,13 @@ type: function(file, uploadInfo?)
 A function that cancels the file upload.
 
 ##### param(file): File
-<!-- Description goes here -->
+The file that is uploaded.
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxFileManager/5 File System Providers/Custom/1 Configuration/abortFileUpload.md]:
```suggestion
The file that is being uploaded.
```

File: api-reference/10 UI Widgets/dxFileManager/5 File System Providers/Custom/1 Configuration/createDirectory.md [hunk 5]
```diff
@@ -7,13 +7,13 @@ type: function(parentDirectory, name)
 A function that creates a folder.
 
 ##### param(parentDirectory): FileSystemItem
-<!-- Description goes here -->
+The directory where a new subdirectory should be created.
 
 ##### param(name): String
-<!-- Description goes here -->
+The name of the new directory.
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxFileManager/5 File System Providers/Custom/1 Configuration/createDirectory.md:13]: So what are we creating: a folder, a directory, or a subdirectory?

File: api-reference/10 UI Widgets/dxFileManager/5 File System Providers/FileSystemProviderBase/3 Methods/abortFileUpload().md [hunk 6]
```diff
@@ -15,7 +15,7 @@ id: FileSystemProviderBase.abortFileUpload()
 <!-- Description goes here -->
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxFileManager/5 File System Providers/FileSystemProviderBase/3 Methods/abortFileUpload().md]: There are empty descriptions in this file and those that follow it.

File: api-reference/10 UI Widgets/dxFileManager/5 File System Providers/Custom/1 Configuration/uploadFileChunk.md [hunk 7]
```diff
@@ -7,13 +7,14 @@ type: function(file, uploadInfo)
 A function that uploads a file in chunks.
 
 ##### param(file): File
-<!-- Description goes here -->
+The file that is uploaded.
```
Comment (RomanTsukanov) [inline api-reference/10 UI Widgets/dxFileManager/5 File System Providers/Custom/1 Configuration/uploadFileChunk.md]:
```suggestion
The file that is being uploaded.
```

**Review by RomanTsukanov — CHANGES_REQUESTED** [review]

File: api-reference/10 UI Widgets/dxFileManager/5 File System Providers/FileSystemProviderBase/3 Methods/abortFileUpload().md (hunk 6 above)
Comment (AnnaKondratova) [reply api-reference/10 UI Widgets/dxFileManager/5 File System Providers/FileSystemProviderBase/3 Methods/abortFileUpload().md]: they will be described in the next PR

**Comment by AnnaKondratova** [comment]
Reminder about this [PR](https://github.com/DevExpress/devextreme-documentation/pull/307).

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**